src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -169,11 +178,18 @@ public boolean link() throws IOException, InterruptedException {
         }
         if (!useJavaFX) return answer;
 
-        answer.addAll(linuxfxlibs);
+        if (projectConfiguration.getTargetTriplet().getArch().equals(Constants.ARCH_AARCH64)) {
```

Comment:
We can refactor `projectConfiguration.getTargetTriplet().getArch().equals(Constants.ARCH_AARCH64)`, at least to a local variable. 
We already have `crossCompile`, which might be enough?

Reply:
crossCompile is not enough, as it is also true in the opposite case (running on linux-arch64, compiling for AMD64)
I'll add an instance variable